### PR TITLE
improve rank table

### PR DESCRIPTION
**Pull request by Tolu-Mals**
# Summary
Closes https://github.com/EveripediaNetwork/issues/issues/1727

In this pr I
- added ability to toggle the ordering of tokens/nft on ranking table
- made the tokens/nfts be ordered by market cap by default
- fixed unwanted scrolling when switching tabs on the rank table
- removed rank table params on home page

### CODE REVIEW

**Review by Simplypheyie96 — COMMENTED** [review]
@Tolu-Mals I think we should add an hover state for the marketcap table header so that people can know it's clickable. 
image

**Comment by Tolu-Mals** [comment]
> @Tolu-Mals I think we should add an hover state for the marketcap table header so that people can know it's clickable. image

I don't think a hover state solves the problem, the cursor already changes to a pointer when we hover on the button, so that should be enough for when a user is hovering on the button.

Maybe if we styled it in such a way that suggests it's clickable though

**Comment by Simplypheyie96** [comment]
> > @Tolu-Mals I think we should add an hover state for the marketcap table header so that people can know it's clickable. image
> 
> I don't think a hover state solves the problem, the cursor already changes to a pointer when we hover on the button, so that should be enough for when a user is hovering on the button.
> 
> Maybe if we styled it in such a way that suggests it's clickable though

how do you mean?

**Comment by Tolu-Mals** [comment]
> > > @Tolu-Mals I think we should add an hover state for the marketcap table header so that people can know it's clickable. image
> > 
> > 
> > I don't think a hover state solves the problem, the cursor already changes to a pointer when we hover on the button, so that should be enough for when a user is hovering on the button.
> > Maybe if we styled it in such a way that suggests it's clickable though
> 
> how do you mean?

We could style either the header or just header icon to look like a button

**Comment by Tolu-Mals** [comment]
> maybe this icon would be better. I believe it would look a lot better to have a hover state for that instead of a button that would stick out on the table. @Tolu-Mals image

Alright, let's try that

**Comment by Tolu-Mals** [comment]
Let me know when I can check the Figma

**Comment by Simplypheyie96** [comment]
> Let me know when I can check the Figma

you can now: https://www.figma.com/file/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.WIKI?type=design&node-id=15389-139138&mode=design&t=irEl8fFUBDrvavNH-11

**Review by Softdev1 — APPROVED** [review]

**Review by AshakaE — APPROVED** [review]

**Comment by Simplypheyie96** [comment]
The icon is yet to be changed @Tolu-Mals

**Comment by Softdev1** [comment]
> The icon is yet to be changed @Tolu-Mals

@Aliiiu  will complete it this morning

**Comment by Aliiiu** [comment]
> The icon is yet to be changed @Tolu-Mals

You can check now

https://iq-wiki-1x0xwhreb-prediqt.vercel.app/

**Review by Simplypheyie96 — APPROVED** [review]

File: src/pages/rank/index.tsx [hunk 1]
```diff
@@ -25,12 +31,41 @@ import { store } from '@/store/store'
...
+export const sortByMarketCap = (
+  order: SortOrder,
+  items: RankCardType[],
+  setOrder: Dispatch<SetStateAction<SortOrder>>,
+) => {
+  const innerItems = [...items]
+
+  try {
+    innerItems.sort((a, b) => {
+      const AMarketCap =
```
[the remark this answers is not in the document]
Comment (Aliiiu) [reply src/pages/rank/index.tsx]: check the function now

**Review by Adebesin-Cell — APPROVED** [review]

**Pull request merged**